control/bdalg.py

```diff
@@ -54,6 +54,7 @@
 """
 
 import numpy as np
+from scipy.signal.ltisys import StateSpace
```

Comment:
Shouldn't this be importing control.StateSpace?

Reply:
hmm, yes, not sre how that got in there. removed.